src/OctoshiftCLI.Tests/bbs2gh/Handlers/GenerateScriptCommandHandlerTests.cs

```diff
@@ -82,7 +82,211 @@ public async Task No_Projects()
         await _handler.Handle(args);
 
         // Assert
-        _mockFileSystemProvider.Verify(m => m.WriteAllTextAsync(It.IsAny<string>(), It.Is<string>(script => TrimNonExecutableLines(script, 9, 0) == "")));
+        _mockFileSystemProvider.Verify(m => m.WriteAllTextAsync(It.IsAny<string>(), It.Is<string>(script => TrimNonExecutableLines(script, 33, 0) == "")));
+    }
+
+    [Fact]
+    public async Task Validates_Env_Vars()
```

Comment:
Do you think that we should have an integration test that runs the PowerShell script as well?

Reply:
All our integration tests run the powershell script